src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.cs

```diff
@@ -15,6 +15,8 @@ namespace System.IO.Compression.Tests
 {
     public partial class ZipFileTest_ConvenienceMethods : ZipFileTestBase
     {
+        public static bool IsNotMacOsHighSierraOrHigher => PlatformDetection.IsNotMacOsHighSierraOrHigher;
```

Comment:
There doesn't seem to be the type overload of `ConditionalFact` that allows you to specify a conditional function from a different class for the version of buildtools being used in rel/2.0.0